migrations/message_store_postgres/content_script_version_7.nim

```diff
@@ -0,0 +1,29 @@
+const ContentScriptVersion_7* =
+  """
+
+-- Create lookup table
+CREATE TABLE IF NOT EXISTS messages_lookup (
+   timestamp BIGINT NOT NULL,
+   messageHash VARCHAR NOT NULL,
+   CONSTRAINT messageIndexLookupTable PRIMARY KEY (messageHash, timestamp)
+  );
+
+-- Put data into lookup table
+INSERT INTO messages_lookup (messageHash, timestamp) SELECT messageHash, timestamp from messages;
```

Comment:
I do not know  how slow this operation is, but since it's doing a batch insert, a way to speed up the insertion could be to create the table without the primary key, and then, after the insertion is complete, do:
```sql
ALTER TABLE messages_lookup ADD CONSTRAINT messageIndexLookupTable PRIMARY KEY (messageHash, timestamp)
```
This should be faster because it eliminates the overhead of maintaining the primary key index during each insert operation.

Reply:
(of course, if the insertion takes just a couple of minutes then doing this optimization is an overkill)

Reply:
cool thanks! I've applied the suggestion in https://github.com/waku-org/nwaku/pull/2933/commits/cb80df4f1b6035040b3fe7328f094553e57448dc